tools/lkl/tests/net.sh

```diff
@@ -43,15 +43,19 @@ fi
 if ! [ -z $DST ]; then
     echo "== RAW socket (LKL net) tests =="
     sudo ip link set dev ${IFNAME} promisc on
-    sudo ./net-test raw ${IFNAME} ${DST} dhcp
+    sudo ./net-test raw ${IFNAME} ${DST} dhcp &
+    sleep 5
+    sudo killall -q net-test && (echo "killed, skipped")
```

Comment:
I don't know how it works. If killall doesn't find net-test, it still return 1 which is considered a failure by the script?